Sources/Components/SafetyElementsView/SafetyElementViewModel.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Copyright © 2020 FINN AS. All rights reserved.
+//
+
+import Foundation
+
+public struct SafetyElementViewModel {
+    let title: String
+    let icon: UIImage
+    let body: String
+    let topLink: LinkButtonViewModel?
+    let bottomLink: LinkButtonViewModel?
```

Comment:
In case we need to debug stuff in the app it'd be great if these were marked as `public` so we can access them :)